judge/chroot-settings.sh.in

```diff
@@ -1,7 +1,7 @@
 #/bin/sh
 
 # Chroot subdirs needed: (optional lib64 only needed for amd64 architecture)
-SUBDIRMOUNTS="etc usr lib bin"
+SUBDIRMOUNTS="etc usr lib bin opt"
```

Comment:
I don't like adding `opt` just because it's a possible place to install software. Admins can add it themselves if they really need it, but let's stick to the bare minimum for security sake.

Reply:
Will revert.